include/roboteam_ai/stp/Play.hpp

```diff
@@ -81,12 +69,35 @@ class Play {
      */
     Play(Play&& other) = default;
 
+    /**
+     * Check if the preconditions of this play are true
+     * @return true if the play is allowed to be started, else false
+     */
+    [[nodiscard]] virtual bool isValidPlayToStart(world_new::World* world) noexcept = 0;
```

Comment:
Reference instead of pointer

Reply:
World is managed so it shouldn't really matter, right?